io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -67,31 +67,12 @@ public:
     */
    std::shared_ptr<TBufferMergerFile> GetFile();
 
-   /** Returns the number of buffers currently in the queue. */
-   size_t GetQueueSize() const;
-
-   /** Returns the number of bytes currently buffered (i.e. in the queue). */
-   size_t GetBuffered() const
-   {
-      return fBuffered;
-   }
-
```

Comment:
Do we need to properly deprecate these methods, or is just removing them fine?

Reply:
We need to properly deprecated those, as they have been used for the user to implement their own blocking mechanism.